Containers/Footer/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Button } from 'react-native';
+
+export default class Footer extends Component {
+
+  render() {
+    return (
+      <View style={styles.container}>
+      </View>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    position: 'absolute',
+    bottom: 0,
```

Comment:
Should this be 'fixed' or 'absolute'?
If we add more info below and need to scroll down, the red bar might need to be fixed?

Reply:
It's currently 0px from the bottom of our App Component which takes up 100% height, so it will work as intended. It seems that currently react-native doesn't support the 'fixed' attribute

Reply:
That's "absolutely" crazy. 
Ok, great. We can use absolute then. Thanks for looking into that.